### PR TITLE
finding duplicate applicants in Lega

### PR DESCRIPTION
**Pull request by ebeshero**
@zme1 @djbpitt Here's my quick and dirty readout of all the duplicate applicants in your minutes and where they appear. Hope this helps!

### CODE REVIEW

File: TEI/meetingMinutes/ebb_dupApplicantRefs.xquery [hunk 1]
```diff
@@ -6,4 +6,4 @@ for $i in $dvApps
 let $match := //list[@type="applicants"]//persName[@ref = $i]
 where count($match) gt 1
 return 
-($i, $match/ancestor::TEI//date/@when/string(),  "&#10;")
+($i, "appears repeatedly here:", string-join($match/ancestor::TEI//date/@when/string(), ', '),  ". &#10;")
```
Comment (ebeshero) [inline TEI/meetingMinutes/ebb_dupApplicantRefs.xquery]: @zme1 I just added some strings to make this a little easier to read quickly. It's basically an index of repeat appearances.

**Comment by zme1** [comment]
The fact that there are that many doesn’t surprise me, because some are mentioned twice in a single list, but I think that some of those duplicates are false-positives. The reason I say that is I got some extra duplicates when I didn’t specify that I was only looking at persName elements that are children of an item element. Otherwise it would also search the notes embedded within the list, inside of which there are some repeated names. I’m sorry I wasn’t more specific on that front...

Le 17 mars 2018 à 23:34, Elisa Beshero-Bondar <notifications@github.com<mailto:notifications@github.com>> a écrit :


@zme1<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fzme1&data=01%7C01%7Czacharyenick%40pitt.edu%7Cfe5df86effa34ab8b0c508d58c811f3e%7C9ef9f489e0a04eeb87cc3a526112fd0d%7C1&sdata=eOvWPcKFZ5aYhZZPlH8hUWL8o54HbduVOQbgAKvn6Q0%3D&reserved=0> @djbpitt<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fdjbpitt&data=01%7C01%7Czacharyenick%40pitt.edu%7Cfe5df86effa34ab8b0c508d58c811f3e%7C9ef9f489e0a04eeb87cc3a526112fd0d%7C1&sdata=Ww4ZnyqKcoWlZtUewxQhKZy6x0W5KzWbjHYE4l4hKFQ%3D&reserved=0> Here's my quick and dirty readout of all the duplicate applicants in your minutes and where they appear. Hope this helps!

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/zme1/toscana/pull/17<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fzme1%2Ftoscana%2Fpull%2F17&data=01%7C01%7Czacharyenick%40pitt.edu%7Cfe5df86effa34ab8b0c508d58c811f3e%7C9ef9f489e0a04eeb87cc3a526112fd0d%7C1&sdata=%2BnBhDyfvFgOI%2FIAAPrIayCgkI8KohmkiQtC%2BYwUsAmc%3D&reserved=0>

Commit Summary

  *   finding duplicate applicants in Lega

File Changes

  *   A TEI/meetingMinutes/ebb_dupApplicantRefs.xquery<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fzme1%2Ftoscana%2Fpull%2F17%2Ffiles%23diff-0&data=01%7C01%7Czacharyenick%40pitt.edu%7Cfe5df86effa34ab8b0c508d58c811f3e%7C9ef9f489e0a04eeb87cc3a526112fd0d%7C1&sdata=KhhfrrhO81gXyui5EsIKfeT5lsNsC0qti8lUVg1WeS8%3D&reserved=0> (9)
  *   A TEI/meetingMinutes/repeatApplicants.txt<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fzme1%2Ftoscana%2Fpull%2F17%2Ffiles%23diff-1&data=01%7C01%7Czacharyenick%40pitt.edu%7Cfe5df86effa34ab8b0c508d58c811f3e%7C9ef9f489e0a04eeb87cc3a526112fd0d%7C1&sdata=6ek3h8Rlz%2FTs%2BQ1Wgw5L8wZ7WY%2Fr95rQpq%2BGd8BNyZk%3D&reserved=0> (16)

Patch Links:

  *   https://github.com/zme1/toscana/pull/17.patch<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fzme1%2Ftoscana%2Fpull%2F17.patch&data=01%7C01%7Czacharyenick%40pitt.edu%7Cfe5df86effa34ab8b0c508d58c811f3e%7C9ef9f489e0a04eeb87cc3a526112fd0d%7C1&sdata=C%2Fj8QpjRcCU9gHC0EzQY8ZtijjNPlBlkWbeoQcoFN3c%3D&reserved=0>
  *   https://github.com/zme1/toscana/pull/17.diff<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fzme1%2Ftoscana%2Fpull%2F17.diff&data=01%7C01%7Czacharyenick%40pitt.edu%7Cfe5df86effa34ab8b0c508d58c811f3e%7C9ef9f489e0a04eeb87cc3a526112fd0d%7C1&sdata=CNuA2SYp%2BZukHzh879FlEvSMwdCs1FCqQFhmhlsuX7o%3D&reserved=0>

**Comment by ebeshero** [comment]
@zme1 Ahh! That's easily fixed! Let's see what happens if we adjust the variable...

File: TEI/meetingMinutes/ebb_dupApplicantRefs.xquery [hunk 2]
```diff
@@ -1,9 +1,9 @@
 declare default element namespace "http://www.tei-c.org/ns/1.0";
 
-let $applicants := //list[@type="applicants"]//persName/@ref/string()
+let $applicants := //list[@type="applicants"]/item/persName/@ref/string()
```
Comment (ebeshero) [inline TEI/meetingMinutes/ebb_dupApplicantRefs.xquery]: @zme1 Is this what you're looking for?

File: TEI/meetingMinutes/repeatApplicants.txt [hunk 3]
```diff
@@ -1,16 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>#maffeir appears repeatedly here: 1919-03-18, 1925-03-19, 1925-05, 1925-04 . 
  #pacinio appears repeatedly here: 1919-08-18, 1919-10, 1919-11-06, 1925-05-21, 1925-07 . 
  #cappellinim appears repeatedly here: 1919-11-06, 1925-05-21, 1925-07 . 
- #cinellio appears repeatedly here: 1919-11-06 .
```
Comment (ebeshero) [inline TEI/meetingMinutes/repeatApplicants.txt]: @zme1 I think this removed a few false positives...(pink indicates lines removed from the previous commit). How about the others?

**Pull request merged**